Content.Server/Botany/Components/PlantHolderComponent.cs

```diff
@@ -728,23 +729,21 @@ async Task<bool> IInteractUsing.InteractUsing(InteractUsingEventArgs eventArgs)
 
             var solutionSystem = EntitySystem.Get<SolutionContainerSystem>();
             if (solutionSystem.TryGetDrainableSolution(usingItem, out var solution)
-                && solutionSystem.TryGetSolution(Owner, SoilSolutionName, out var targetSolution))
+                && solutionSystem.TryGetSolution(Owner, SoilSolutionName, out var targetSolution) && _entMan.TryGetComponent(usingItem, out SprayComponent? spray))
             {
-                var amount = FixedPoint2.New(5);
-                var sprayed = false;
+                var amount = FixedPoint2.New(1);
+
+                var sprayed = true;
```

Comment:
Given that sprayed is always true now, you can just remove it.

Reply:
Yeah that makes sense, I changed the popup message to just be for sprays and removed sprayed